client/blocks/comment-button/index.jsx

```diff
@@ -20,51 +19,50 @@ import { getPostTotalCommentsCount } from 'calypso/state/comments/selectors';
 import './style.scss';
 
 function CommentButton( props ) {
-	const { commentCount, href, onClick, showLabel, tagName, target } = props;
+	const { commentCount, href, onClick, size, target } = props;
 	const translate = useTranslate();
 
-	return React.createElement(
-		tagName,
-		omitBy(
-			{
-				className: 'comment-button',
-				href: 'a' === tagName ? href : null,
-				onClick,
-				target: 'a' === tagName ? target : null,
-			},
-			( prop ) => prop === null
-		),
-		<Gridicon icon="comment" size={ props.size } className="comment-button__icon" />,
-		<span className="comment-button__label">
-			{ commentCount > 0 && <span className="comment-button__label-count">{ commentCount }</span> }
-			{ showLabel && commentCount > 0 && (
+	return (
+		<a className="comment-button" href={ href } onClick={ onClick } tabIndex="0" target={ target }>
+			<Gridicon aria-hidden="true" icon="comment" size={ size } className="comment-button__icon" />
+			<span className="comment-button__label">
+				{ commentCount > 0 && (
+					<span className="comment-button__label-count">{ commentCount }</span>
+				) }
 				<span className="comment-button__label-status">
-					{ translate( 'Comment', 'Comments', {
-						context: 'noun',
-						count: commentCount,
-					} ) }
+					{ commentCount > 0 && (
+						<Fragment>
+							{ translate( 'Comment', 'Comments', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **164** times:
`translate( 'Comment', { context: 'noun'} )` *ES Score: 12*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/reader-comments-button-a11y)


:information_source: This string doesn't exist elsewhere without a **context** (or with a different one). You can probably drop the context. If needed, you can use a translator comment instead to clarify meaning.